app/src/main/java/com/infomaniak/mail/utils/extensions/AttachmentExtensions.kt

```diff
@@ -185,8 +184,12 @@ object AttachmentExtensions {
                 SentryLog.d(ATTACHMENT_TAG, "When removing uploaded attachment, we found (uuids to localUris): $uuidToLocalUri")
                 SentryLog.d(ATTACHMENT_TAG, "Target uploadLocalUri is: $uploadLocalUri")
 
-                // The API version of an Attachment doesn't have the `uploadLocalUri` & `localUuid`, so we need to back them up.
-                remoteAttachment.backupLocalData(oldAttachment = this@updateLocalAttachment)
+                remoteAttachment.backupLocalData(oldAttachment = this@updateLocalAttachment, UploadStatus.FINISHED)
+
+                SentryLog.d(ATTACHMENT_TAG, "Uploaded attachment name: ${remoteAttachment.name}")
+                SentryLog.d(ATTACHMENT_TAG, "Uploaded attachment uuid: ${remoteAttachment.uuid}")
+                SentryLog.d(ATTACHMENT_TAG, "Uploaded attachment localUuid: ${remoteAttachment.localUuid}")
+                SentryLog.d(ATTACHMENT_TAG, "Uploaded attachment uploadLocalUri: ${remoteAttachment.uploadLocalUri}")
```

Comment:
We might leak some personal information if we log `uploadLocalUri` right?

Reply:
`name` was removed.